server/genserver.go

```diff
@@ -186,11 +197,24 @@ func jsonMarshalNoError(i interface{}) string {
 {{end}}
 `
 
-func generateHandlers(packageName, packagePath string, s *spec.Swagger, paths *spec.Paths) error {
+func extractModuleNameAndVersionSuffix(packageName, outputPath string) (moduleName, versionSuffix string) {
```

Comment:
This function seems to appear exactly in 3 or more places?

Reply:
Yep, should have written a TODO for that one. I moved it into utils.